Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -21,8 +21,8 @@ class Tests < AbstractCommand
         switch "--online",
                description: "Include tests that use the GitHub API and tests that use any of the taps for " \
                             "official external commands."
-        switch "--byebug",
-               description: "Enable debugging using byebug."
+        switch "--debug",
```

Comment:
Yeah, I thought that `--debug` can trigger both (ie, interactive debugging if a breakpoint is set, "normal" `odebug` things otherwise) without the cognitive load of having to remember that interactive debugging is another option.